app/src/main/java/nerd/tuxmobil/fahrplan/congress/changes/ChangeStatistic.kt

```diff
@@ -0,0 +1,47 @@
+package nerd.tuxmobil.fahrplan.congress.changes
+
+import info.metadude.android.eventfahrplan.commons.logging.Logging
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+
+/**
+ * Statistic about lecture changes (canceled, changed, new).
+ */
+class ChangeStatistic @JvmOverloads constructor(
+
+        val lectures: List<Lecture>,
+        val logging: Logging = Logging.get()
+
+) {
+
+    /**
+     * Returns how many lectures are marked as [canceled][Lecture.changedIsCanceled].
+     */
+    fun getCanceledLecturesCount() = lectures
+            .count { it.changedIsCanceled }
+            .also { log("$it canceled lectures") }
+
+    /**
+     * Returns how many lectures are marked as [changed][Lecture.isChanged].
+     */
+    fun getChangedLecturesCount() = lectures
+            .count { it.isChanged }
+            .also { log("$it changed lectures") }
+
+    /**
+     * Returns how many lectures have are marked as [new][Lecture.changedIsNew].
```

Comment:
typo: -have

Reply:
Fixed in e5337468b8d9380c77a4931e6dd748e7410eb429.